test/appium/tests/atomic/account_management/test_wallet_management.py

```diff
@@ -197,3 +197,18 @@ def test_filter_transactions_history(self):
                 pytest.fail('Failed transactions are not filtered')
             details.back_button.click()
         self.verify_no_errors()
+
+    @marks.testrail_id(5381)
+    @marks.high
+    @marks.skip
+    def test_user_can_see_all_own_assets_after_account_recovering(self):
+        account = 'record sphere illegal section blame swing wreck estate celery same excite summer'
```

Comment:
You can use here one of the wallet users from the `/users.py` file or create a new one there

Reply:
@yevh-berdnyk 
yes, I know. But this exact user is waiting for test KudosToken. I requested it.
Not a problem to move it to users.py but then it will come reusable.

Reply:
@yevh-berdnyk 
did as discussed